framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -720,12 +726,14 @@ public boolean removeAnnotationInHierarchy(AnnotationMirror a) {
    *
    * @param a an annotation from the same qualifier hierarchy
    * @return if an annotation was removed
+   * @deprecated TODO what should be called?
```

Comment:
What should be called instead?

Reply:
This is never called, and I can't see how it is useful, so I think it should just be removed.  I've update the Javadoc to reflect that.